plugins/formatter-prettier/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "webpack --watch",
     "build": "webpack",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
I think we can remove it

Reply:
let's keep it since we will need to add some tests here..